restapi/src/main/java/io/stargate/web/docsapi/service/JsonDocumentShredder.java

```diff
@@ -165,6 +165,13 @@ private void processObjectNode(
             field -> {
               String fieldName = field.getKey();
 
+              if (fieldName.isEmpty()) {
+                String msg =
+                    "JSON objects containing empty field names are not supported at the moment.";
```

Comment:
nit: maybe just "not supported" since otherwise we are implying there's a chance/plan for changing.
If users want this to be supported they can of course request it.